docs/pages/access-controls/sso/gitlab.mdx

```diff
@@ -4,60 +4,91 @@ description: How to configure Teleport access using GitLab for SSO
 h1: Teleport SSO Authentication with GitLab
 ---
 
-## How to use GitLab as a single sign-on (SSO) provider with Teleport
-
 This guide will cover how to configure [GitLab](https://www.gitlab.com/) to issue
-SSH credentials to specific groups of users. When used in combination with role
+credentials to specific groups of users. When used in combination with role
 based access control (RBAC), it allows administrators to define policies
 like:
 
-- Only members of "DBA" group can SSH into machines running PostgreSQL.
+- Only members of "DBA" group can access PostgreSQL databases.
 - Only members of "ProductionKubernetes" can access production Kubernetes clusters
 - Developers must never SSH into production servers.
 
 ## Prerequisites
 
-- At least two groups in GitLab with users assigned.
-- Teleport role with access to maintaining `oidc` resources.  This is available in the default `editor` role.
+- At least two groups in GitLab with users assigned. In our examples below,
+  we'll assume a group named `company` with two subgoups, `admin` and `dev`.
+- Teleport role with access to maintaining `oidc` resources.  This is available
+  in the default `editor` role.
 
 (!docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
 - (!docs/pages/includes/tctl.mdx!)
 
-## Enable default OIDC authentication
-
-(!docs/pages/includes/enterprise/oidcauthentication.mdx!)
-
 ## Configure GitLab
 
-You should have at least one group configured in GitLab to map to Teleport roles. In this example we use the names `gitlab-dev` and `gitlab-admin`.  Assign users to each of these groups.
-
-1. Create a Application in one of your Groups that will allow using GitLab as a OAuth provider to Teleport.
+You should have at least one group configured in GitLab to map to Teleport roles.
+In this example we use the names `gitlab-dev` and `gitlab-admin`.  Assign users
+to each of these groups.
 
-Settings
+1. Create a Application in one of your Groups that will allow using GitLab as a
```

Comment:
Should "Application" and "Groups" be proper nouns here?

Reply:
No, the capitalization relates to the way they are presented in the GitLab nav menu. We could emphasize this with bold, or a screenshot, or just remove the caps.

Reply:
I would be more explicit that "a Application" and "Groups" refer to the GitLab nav menu, otherwise this just looks like ungrammatical English. I think bolding the text and describing the relevant parts of the UI would be good.